modules/ols-attaching-a-resource-object-to-your-query.adoc

```diff
@@ -0,0 +1,29 @@
+// This module is used in the following assemblies:
+// ols-using-openshift-lightspeed.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ols-attaching-a-resource-object-to-your-query_{context}"]
+= Attaching a resource object to your query
+
+This procedure explains how to attach a resource object to provided additional context for your chat query. 
+
+.Procedure
+
+. Navigate to a supported resource in the {ocp-product-title} web console. For example, click *Workloads* -> *Pods* and then click the name of a pod.
++
+[NOTE]
+====
+If you bring up the user interface using any of the *Ask OpenShift Lightspeed* menu options, the attachment option is present.
+====
+
+. Click the {ols-offical} icon in the lower-right corner of the screen.
+
+. Click the plus icon in the {ols-offical} natural language interface to attach an object.
+
+. Select the resource object to attach to the query.
```

Comment:
there are ~4 types of attachments:
1. full yaml
2. yaml status block (for some objects)
3. logs (for pods)
4. alerts (for pods)

Should we document each individually?

Reply:
@thoraxe I'm not sure if creating an example for each provides much value add. 

I can certainly mention the 4 types of attachments to provide some perspective.

Reply:
That should be "Events" rather than "alerts"

Here's the attach options for a pod:
![attach-options](https://github.com/user-attachments/assets/bc09cba3-98dc-49c7-b944-60e6d22fd144)

"Events" and "Logs" are only shown for pods.

"YAML" and "YAML status only" are shown for all supported resources except Alerts.

For Alerts, we show the single option "Alert", which is actually the full YAML for the alert (same as "YAML" on the other resource pages).

Reply:
@kyoto sorry, I think I'm going to have a quick chat with you about this just to make sure I'm following correctly.